src/test/java/io/vlingo/http/resource/TestDispatcher.java

```diff
@@ -17,7 +17,7 @@
 
   public TestDispatcher(final Resources resources) {
```

Comment:
I would suggest passing the logger in constructor. And in `io.vlingo.http.resource.ResourceTestFixtures`, getting it like this `new TestDispatcher(resources, world.defaultLogger())`

Reply:
@alexguzun sure, that sounds good, will do. Thank you